test/sequential/test-cluster-net-listen-ipv6only-rr.js

```diff
@@ -16,37 +15,40 @@ const host = '::';
 const WORKER_ACCOUNT = 3;
 
 if (cluster.isMaster) {
-  const workers = new Map();
+  const workers = [];
   let address;
 
-  const countdown = new Countdown(WORKER_ACCOUNT, () => {
-    // Make sure the `ipv6Only` option works.
+  for (let i = 0; i < WORKER_ACCOUNT; i += 1) {
+    const myWorker = new Promise((resolve) => {
+      const worker = cluster.fork().on('exit', common.mustCall((statusCode) => {
+        assert.strictEqual(statusCode, 0);
+      })).on('listening', common.mustCall((workerAddress) => {
+        if (!address) {
+          address = workerAddress;
+        } else {
+          assert.deepStrictEqual(workerAddress, address);
+        }
+        resolve(worker);
+      }));
+    });
+
+    workers.push(myWorker);
+  }
+
+  Promise.all(workers).then((resolvedWorkers) => {
```

Comment:
Likewise, the then handler should be wrapped in a `common.mustCall()`

Reply:
Likewise, done!